R/app_template.R

```diff
@@ -122,58 +132,79 @@ shinyAppTemplate <- function(path = NULL, examples = "default")
     system.file("app_template", path, package = "shiny")
   }
 
-  mkdir_p <- function(path) {
-    dir.create(path, showWarnings = FALSE, recursive = TRUE)
+  mkdir <- function(path) {
+    if (!dirExists(path)) {
+      message("Creating ", ensure_trailing_slash(path))
+      if (!dryrun) {
+        dir.create(path, recursive = TRUE)
+      }
+    }
+  }
+
+  copy_file <- function(from, to, is_template = FALSE) {
```

Comment:
Can maybe move the `is_template` logic into the function as it's consistent across the board

Reply:
or make the `is_template` logic a helper function